### ISSUE TITLE
Cinderfruit has really high saturation

### ISSUE THREAD

**Issue by VeggeieDude** [raltsmc/desolation#33]
Heya! I saw that Cinderfruit has 33 saturation points lol, is that intended?

I'm using fabric version 1.3.0 on 1.20.1

Here is a screenshot
![2023-12-02_02 55 05](https://github.com/raltsmc/desolation/assets/90440365/dfa5e3f3-bebe-40c3-b60f-dc595b6c0c28)

**Comment by gniftygnome** [comment]
Yeah, looking at the source it's 5 hunger x 6.5 saturation modifier = 32.5 saturation points.  The highest-saturation food in vanilla is the golden carrot at 6 x 1.2 = 7.2.  I'm not sure exactly what RaltsMC intended for the Cinderfruit.  It's found in a really high-risk biome, so if it were a limited-availability item, crazy-high saturation might make some sense.  However, it's easily farmed (the main limiting factor is the player must gather charred soil to farm it on).

I think a more appropriate saturation modifier might be 2.0, which would make it worth 10 points.  This is high, but within the same order of magnitude as other high-saturation foods.  It would still be worth farming for the saturation (or for the fire resistance).  (Sooner or later somebody is going to come whining about how they can swim in lava if they eat Cinderfruit...)
